FileSystem/FileMapperMocker.js

```diff
@@ -10,28 +25,490 @@ var USE_FILEMAPPER_MOCKER = true;
 var VBAFileMapperMocker = {
```

Comment:
The old fileMapper is not removed yet to avoid conflict issues, will be removed eventually

Reply:
Update: code is removed